src/core/api/index.js

```diff
@@ -60,15 +61,18 @@ export function callApi({
   const apiURL = `${API_BASE}/${endpoint}/${queryString}`;
 
   return fetch(apiURL, options)
-    .then((response) => response.json().then(
-      (jsonResponse) => ({ response, jsonResponse }),
-      (error) => {
-        log.warn('Could not parse response as JSON:', error);
-        return response.text().then((textResponse) =>
-          ({ response, jsonResponse: { text: textResponse } })
+    .then((response) => {
+      return response.clone().json()
+        .then(
+          (jsonResponse) => ({ response, jsonResponse }),
+          (error) => {
```

Comment:
Ahh, gotcha. Yeah the fetch API is a bit odd. I need to learn the async/await syntax, though it def looks nicer 😄